### PR TITLE
Fix/move secrets to gcloud secret manager

### LINKED ISSUE #6: Migrate secret management to Google Secret Manager instead of Firestore
At this point, the app is initialized under a subprocess, in which the main process is only used to load secrets from firestore.

This process is weird, and difficult to understand, so, It´s needed to move this logic to Google Secret Manager and adapt build process via Github Actions

### PR DESCRIPTION
**Pull request by ernestognw**
Moved secret logic to GCloud Secret Manager - Fixes #6 

A previous investigation recommended to put secrets inside Firestore, and then consume them using the programming language in which you're deploying.

The problem was that due to concurrency issues on how NodeJS works, it was required to create a subprocess, which is pretty difficult and not clear for other contributors.

This commit changes that behaviour to a more standard one relying on Github Actions and [this guide](https://github.com/GoogleCloudPlatform/github-actions/blob/master/get-secretmanager-secrets/README.md)

### General updates
- **Comments**: Rewrote comments accordingly
- **README**: Added a section about how to add new env variables and remove manual deployment since it is not encouraged
- **Deployment**: Added steps to authenticate and retrieve secrets from secret manager
- **Build**: Update build to use GIthub Actions environmental variables set.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,27 +28,22 @@ After that, follow the next steps:
...
 
-In order to deploy the project to Google Cloud App Engine, you'll need a project with the following pre-configured APIs and resources:
+In order to deploy the project to Google Cloud App Engine, you'll need a GCloud project with the following pre-configured APIs and resources:
 
 - A Cloud SQL instance running PostgreSQL
 - An App Engine setup for a Node JS service
+- Big Query enabled in your GCloud Project
 
-Open your datastore in Google Cloud, create a collection named `secrets`, then create an entry with the database variables according to [the official docs](https://cloud.google.com/sql/docs/postgres/connect-app-engine-standard?hl=es-419#node.js).
+We use GCloud Secret Manager to store secret keys needed, so, you'll need to add every environmental variable specified in `.env.example` inside the Secret Manager of your GCloud project except for `NODE_ENV` and `PORT` and client side env variables. These variables are set by App Engine by default or directly not needed.
```
Comment (livvielin) [inline README.md:39]: I'm not sure I'm parsing the last sentence here (These variables...). Is the word "directly" needed?

File: README.md [hunk 2]
```diff
@@ -59,6 +54,16 @@ To test the project, you can run:
     - `yarn test:server`
     - `yarn test:client`
 
+# Adding environmental variables
```
Comment (livvielin) [inline README.md:57]: nit: environment?

Here and below

File: README.md (hunk 1 above)
Comment (ernestognw) [reply README.md]: I made a bad spanish-english translation haha
Done!

File: README.md (hunk 2 above)
Comment (ernestognw) [reply README.md]: Done! Thanks

**Comment by ernestognw** [comment]
@livvielin Great! Nits are solved, so I'm merging this

**Pull request merged**